Applications/VGG/jni/main.cpp

```diff
@@ -183,5 +191,9 @@ int main(int argc, char *argv[]) {
     return 1;
   }
 
+#ifdef PROFILE
+  std::cout << *listener;
```

Comment:
How about using #1978  method ?

Reply:
This is same commit which merged. Once the #1978 is merged, I will rebase this.